### PR TITLE
Add a logo to top

### PR DESCRIPTION
**Pull request by nicolasog12**
Add logo to top of webpage - testing

### CODE REVIEW

**Comment by nicolasog12** [comment]
I'm testing it and checking for bugs once it deploys to vercel

**Comment by nicolasog12** [comment]
Ok i found a bug

**Comment by nicolasog12** [comment]
11:48 est - still wating

File: css/index.css [hunk 1]
```diff
@@ -3,6 +3,15 @@ body {
   color: #c9c9c9;
   font-family: Arial, Helvetica, sans-serif;
 }
+header {
+  position: sticky;
+  top: 0;
+  background-color: #1c1c1c;
+  border-bottom: 1px solid black;
```
Comment (3kh0) [inline css/index.css:10]: Make this 0px to hide the border

File: css/projects.css [hunk 2]
```diff
@@ -8,6 +8,15 @@ body {
   background-repeat: no-repeat;
   background-size: cover;
 }
+header {
+  position: sticky;
+  top: 0;
+  background-color: #1c1c1c;
+  border-bottom: 1px solid black;
```
Comment (3kh0) [inline css/projects.css:15]: Make this 0px to hide the border

**Review by 3kh0 — COMMENTED** [review]
It might be a good idea to replace the main text `3kh0.github.io` with the image, see how that looks. See if you can get rid of the black lines as well

image

**Comment by nicolasog12** [comment]
_preparing for reviews..._

File: css/projects.css (hunk 2 above)
Comment (nicolasog12) [reply css/projects.css:15]: ok

**Review by 3kh0 — COMMENTED** [review]
It might be a good idea to replace the main text 3kh0.github.io with the image, see how that looks.

**Comment by nicolasog12** [comment]
> It might be a good idea to replace the main text 3kh0.github.io with the image, see how that looks.

alright workin on it

**Comment by nicolasog12** [comment]
done, waiting for vercel deployment.

**Comment by nicolasog12** [comment]
waiting for deploy

**Comment by nicolasog12** [comment]
ok theres still a bug

**Comment by nicolasog12** [comment]
Im getting an annoying not centering bug, im just gonna keep tryna fix it

**Comment by nicolasog12** [comment]
Ok so I've tried everything I can to fix it and it's obviously either my bad coding or the thing (probably the first one) so I just made the title font Raleway and the color gold, so it look similar.

**Review by 3kh0 — CHANGES_REQUESTED** [review]
image

remove the logo on the top, add the logo to the 3kh0.github.io text bit right above the splash text

**Comment by nicolasog12** [comment]
> image
> 
> remove the logo on the top, add the logo to the 3kh0.github.io text bit right above the splash text

so just replace the title text with the logo image

**Comment by 3kh0** [comment]
yes

**Comment by nicolasog12** [comment]
ill work on it

**Comment by nicolasog12** [comment]
btw if you wanted to know the website that made the logo its just
https://brandmark.io

**Comment by nicolasog12** [comment]
where is the vercel deploy???

File: index.html [hunk 3]
```diff
@@ -26,12 +26,20 @@
     />
   </head>
   <body>
-    <h1 id="title">
-      3kh0.github.io<br />
-      <span id="subtitle">...</span>
-    </h1>
```
Comment (3kh0) [inline index.html:32]: Put the image within the `<h1>` tag

**Review by 3kh0 — CHANGES_REQUESTED** [review]
Please add spacing on the top

image

File: index.html (hunk 3 above)
Comment (nicolasog12) [reply index.html:32]: alright

**Review by 3kh0 — CHANGES_REQUESTED** [review]
The logo at the top needs some padding from the top, and the splash text needs to be centered

**Comment by nicolasog12** [comment]
> The logo at the top needs some padding from the top, and the splash text needs to be centered

The splash text is centered in the css, I can't figure out why its not working

**Pull request closed without merging**